src/main/java/com/rakovets/course/java/core/practice/looping_statements/Task07.java

```diff
@@ -47,6 +47,13 @@ public static void main(String[] args) {
     static String generateTotalPriceList(int startNumberItems, double startPriceAllItems, int differentialNumberItems, int sizeTotalPrice) {
         //TODO
         // Код, решающий задачу пишем ниже, при этом используя параметры метода
-        return null;
+        StringBuilder stringBuilder = new StringBuilder();
```

Comment:
"Violation: Contains code not related to this task
Solution: Remove code not related to this task"